Tests/UnitTests/Attribution/AttributionPosterTests.swift

```diff
@@ -57,6 +55,14 @@ class AttributionPosterTests: TestCase {
                                               attributionFetcher: attributionFetcher,
                                               subscriberAttributesManager: subscriberAttributesManager)
         resetAttributionStaticProperties()
+        backend.stubbedPostAdServicesTokenCompletionResult = .success(())
+    }
+
+    override func tearDown() {
+        super.tearDown()
```

Comment:
Nit: typically this is called at the end of the `tearDown` (Initialize the parent class first, and tear it down last).